merlin/io/__init__.py

```diff
@@ -13,8 +13,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
 # flake8: noqa
+
+from merlin.config import validate_dask_configs
+
+validate_dask_configs()
```

Comment:
@benfred @jperez999 - It doesn't seem like I can add a top-level `merlin/__init__.py` file to perform the dask-config validation without messing up the namespace for libraries like `merlin.dataloaders`.

What do you think about the new `merlin.config` module I added in this PR?

As far as I can tell, we can be sure query-planning is disabled across merlin/nvt if we add this `validate_dask_configs()` call at the top of `merlin/core/__init__.py`, `merlin/io/__init__.py`, and `merlin/dag/__init__.py`. All internal merlin code using dask will need to import one of these modules first. Therefore, we only need to worry about the user importing `dask.dataframe` before importing something from merlin, and I think `validate_dask_configs` raises a suitable error message when this happens.

Reply:
This seems fine. Is there a way for us to test this works, right now? Or do we need to wait until we start using 24.06+ cudf/dask packages?